src/sass/_products.scss

```diff
@@ -0,0 +1,361 @@
+                }
+            }
+        }
+        &:nth-child(3n) {
+            background-color: getColor('prod_bg_third');
+            &:before {
+                transition: transform 275ms 0s ease-in-out;
+                content: '';
+                display: inline-block;
+                background-image: url(../images/product_ice3_mob.png);
+                background-size: 249px 252px;
+                width: 249px;
+                height: 252px;
+                position: absolute;
+                top: -115px;
+                left: 50%;
+                margin-left: -124.5px;
+                @media (min-device-pixel-ratio: 2),
+                    (min-resolution: 192dpi),
+                    (min-resolution: 2dppx) {
+                    background-image: url(../images/product_ice3_mob@2x.png);
+                }
+                @media screen and (min-width: 768px) {
+                    background-size: 220px 226px;
+                    width: 220px;
+                    height: 226px;
+                    top: -100px;
+                    left: 50%;
+                    margin-left: -110px;
+                    background-image: url(../images/product_ice3_tab.png);
+                    @media (min-device-pixel-ratio: 2),
+                        (min-resolution: 192dpi),
+                        (min-resolution: 2dppx) {
+                        background-image: url(../images/product_ice3_tab@2x.png);
+                    }
+                }
+                @media screen and (min-width: 1200px) {
+                    background-size: 335px 335px;
+                    width: 335px;
+                    height: 335px;
+                    top: -200px;
+                    left: 50%;
+                    margin-left: -165.5px;
+                    background-image: url(../images/product_ice3_desk.png);
+                    @media (min-device-pixel-ratio: 2),
+                        (min-resolution: 192dpi),
+                        (min-resolution: 2dppx) {
+                        background-image: url(../images/product_ice3_desk@2x.png);
+                    }
+                }
+            }
+        }
+        @media screen and (min-width: 768px) {
+            flex-basis: calc(100% / 3);
+            padding: 152px 11px 40px;
+            &:not(:last-child) {
+                margin-right: 20px;
+                margin-bottom: 0px;
+            }
+        }
+        @media screen and (min-width: 1200px) {
+            padding: 163px 44px 50px;
+            &:not(:last-child) {
+                margin-right: 24px;
+                margin-bottom: 0px;
+            }
+        }
+        &:hover,
+        &:focus {
+            &:before {
+                transform: rotateY(360deg);
+            }
+            box-shadow: 1px 1px 1px rgba(0, 0, 0, 0.12), 2px 6px 6px rgba(0, 0, 0, 0.06),
+                1px 8px 6px rgba(0, 0, 0, 0.16);
+        }
+    }
+    // Product card content styles
+    &__title {
```

Comment:
&__title {
        color: getColor('white-text');
        text-transform: uppercase;
        font-size: 20px;
        line-height: 1.15;
        &:after {
            content: '...';
            display: block;
            margin: 30px auto;
            font-size: 20px;
        }

        @media screen and (min-width: 1200px) {
            font-size: 30px;
            line-height: 1.133;
        }

Reply:
?